io/src/main/java/org/commonjava/maven/ext/io/PomIO.java

```diff
@@ -101,13 +100,14 @@
             // the plugin versions set inside profiles...so they're not entirely
             // raw.
             Model raw;
+
             try ( InputStream in = new FileInputStream( pom ) )
             {
                 raw = new MavenXpp3Reader().read( in );
             }
             catch ( final IOException | XmlPullParserException e )
             {
-                throw new ManipulationException( "Failed to build model for POM: %s.\n--> %s", e, pom, e.getMessage() );
+                throw new ManipulationException( "Failed to build model for POM: %s.%n--> %s", e, pom, e.getMessage() );
```

Comment:
More %n

Reply:
It should be OK inside an exception message as I see it using other formatting codes in there. I will verify.

Reply:
Huh. I'd completely fogotten that ManipEx had that .... lets discuss that offline.